### PR TITLE
Extended the dataset schema 

### PR DESCRIPTION
**Pull request by igarousi**

### CODE REVIEW

File: schema/dataset-schema.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+``` json
+{
+    "@context" : "https://schema.org/",
+    "@type" : "Dataset",
+    "name":"Fiber-optic distributed temperature sensing data collected in mine-impacted streams near Silverton, Colorado in September 2019 and 2021 (ver. 2.0, May 2022)",
+    "url":"https://www.sciencebase.gov/catalog/item/626b086bd34e76103cd183c5",
+    "identifier":"",
+    "description":"This dataset contains fiber-optic distributed temperature sensing (FO-DTS) data collected along the streambed interface of two streams named Cement Creek and California Gulch Creek, as well as the Animas River, located near Silverton Colorado. ",
+
+    ...
```
Comment (Castronova) [inline schema/dataset-schema.md:28]: Remove the ellipses  (...) from the document. These were used in the prior version but we decided to remove them in core.md and we should do the same here.

**Review by Castronova — CHANGES_REQUESTED** [review]
Looks good, just a few suggestions.

File: schema/dataset-schema.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+for **required** and `0,1` or `0+` for **optional** in the Cardinality column of the table below. 
+
+|Property|Class|Expected Type|Cardinality|Description|
+|---|---|---|---|---|
+|[`Schema:distribution`](https://schema.org/distribution)| Dataset| DataDownload | 1+ | A downloadable form of this dataset, at a specific location, in a specific format. This property can be repeated if different variations are available. There is no expectation that different downloadable distributions must contain exactly equivalent information (see also DCAT on this point). Different distributions might include or exclude different subsets of the entire dataset, for example.|
```
[the remark this answers is not in the document]
Comment (Castronova) [reply schema/dataset-schema.md:8]: I looked at core again and noticed that the format of `Schema:<property name>` is consistent with what we have. Ignore this comment.

[the remark this answers is not in the document]
Comment (igarousi) [reply schema/dataset-schema.md]: Actually, your comment is valid. Links in the table should refer to the subsections in the markdown, not to Schema.org.

File: schema/dataset-schema.md (hunk 1 above)
Comment (igarousi) [reply schema/dataset-schema.md:28]: Done!

File: schema/dataset-schema.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    ...
+       
+    "variableMeasured": {
+        "@type": "PropertyValue",
+        "name": "Streambed interface temperature values",
+        "unitText": "degC"
+    }
+}
+```
+
```
[the remark this answers is not in the document]
Comment (igarousi) [reply schema/dataset-schema.md:45]: Good point! I added the example.

**Review by Castronova — APPROVED** [review]

**Pull request merged**